libgalois/include/katana/analytics/bfs/bfs.h

```diff
@@ -95,9 +95,7 @@ struct KATANA_EXPORT BfsStatistics {
   /// The number of nodes reachable from the source node.
   uint64_t n_reached_nodes;
   /// The maximum distance across all nodes.
-  uint32_t max_distance;
-  /// The average distances on visited nodes from the source node.
-  double average_visited_distance;
+  uint32_t max_parent;
```

Comment:
What is the `max_parent` and why would I care? Maybe just drop this entirely and we can put meaningful stats back in a bit later. 

Also docs need updating here.

Reply:
Hehe yeah, let me remove this! Yeah I am gonna update documents.

Reply:
Agree. Not useful.